src/pages/distroSettings/tabs/HostTab/getFormSchema.tsx

```diff
@@ -0,0 +1,275 @@
+                title: "SSH Option",
+                default: "",
+                minLength: 1,
+              },
+            },
+          },
+        },
+        allocation: {
+          type: "object" as "object",
+          title: "Host Allocation",
+          properties: {
+            version: {
+              type: "string" as "string",
+              title: "Host Allocator Version",
+              oneOf: enumSelect(hostAllocatorVersionToCopy),
+            },
+            roundingRule: {
+              type: "string" as "string",
+              title: "Host Allocator Rounding Rule",
+              oneOf: enumSelect(roundingRuleToCopy),
+            },
+            feedbackRule: {
+              type: "string" as "string",
+              title: "Host Allocator Feedback Rule",
+              oneOf: enumSelect(feedbackRuleToCopy),
+            },
+            hostsOverallocatedRule: {
+              type: "string" as "string",
+              title: "Host Overallocation Rule",
+              oneOf: enumSelect(overallocatedRuleToCopy),
+            },
+            minimumHosts: {
+              type: "number" as "number",
+              title: "Minimum Number of Hosts Allowed",
+              minimum: 0,
+            },
+            maximumHosts: {
+              type: "number" as "number",
+              title: "Maxiumum Number of Hosts Allowed",
+              minimum: 0,
+            },
+            acceptableHostIdleTime: {
+              type: "number" as "number",
+              title: "Acceptable Host Idle Time (s)",
+            },
+            futureHostFraction: {
+              type: "number" as "number",
+              title: "Future Host Fraction",
+              minimum: 0,
+              maximum: 1,
+            },
+          },
+        },
+      },
+    },
+    uiSchema: {
+      setup: {
+        "ui:ObjectFieldTemplate": CardFieldTemplate,
+        bootstrapMethod: {
+          "ui:allowDeselect": false,
+        },
+        communicationMethod: {
+          "ui:allowDeselect": false,
+        },
+        arch: {
+          "ui:allowDeselect": false,
+        },
+        setupScript: {
+          "ui:widget": "textarea",
+        },
+      },
+      sshConfig: {
+        "ui:ObjectFieldTemplate": CardFieldTemplate,
+        sshKey: {
+          "ui:allowDeselect": false,
+        },
+        authorizedKeysFile: {
+          "ui:data-cy": "authorized-keys-input",
+          ...(!hasStaticProvider && { "ui:widget": "hidden" }),
+        },
+        sshOptions: {
+          "ui:addButtonText": "Add SSH option",
+          "ui:descriptionNode": (
+            <>
+              Option keywords supported by <InlineCode>ssh_config</InlineCode>.
+            </>
+          ),
+          "ui:fullWidth": true,
+          "ui:orderable": false,
+          items: {
+            "ui:elementWrapperCSS": css`
+              max-width: unset;
+            `,
+            "ui:placeholder": "ConnectTimeout=10",
+          },
```

Comment:
oops my bad for not specifying! I think I meant something more like this, like in the designs!

```suggestion
          "ui:orderable": false,
          items: {
            "ui:placeholder": "ConnectTimeout=10",
          },
```